### PR TITLE
Add Inside Account Executive job ad

### PR DESCRIPTION
**Pull request by gabrielamatias**
@jesssummerfield , @petehamilton : please review.

### CODE REVIEW

**Comment by gabrielamatias** [comment]
@jesssummerfield , @harrison : please review after changes about comments.

File: app/pages/about/jobs/positions/inside-account-executive-uk.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+
+import React from 'react';
+import JobsPage from '../jobs-page';
+
+export default class InsideAccountExecutive extends React.Component {
+  displayName = 'InsideAccountExecutive'
+
+  render() {
+    return (
+            <JobsPage>
```
Comment (harrison) [inline app/pages/about/jobs/positions/inside-account-executive-uk.js]: Indentation should match the closing tag `</JobsPage>`, also move the stuff bellow to match the `<ul` further down

**Comment by gabrielamatias** [comment]
@jesssummerfield , @harrison : (final?) review after changes.

File: app/pages/about/jobs/positions/inside-account-executive-uk.js (hunk 1 above)
Comment (harrison) [inline app/pages/about/jobs/positions/inside-account-executive-uk.js]: @gabrielamatias nice, usually also indent the whole section two spaces from the previous lines start. So moving the whole start and end of `<JobsPage>`:

```
return (
  <JobsPage>
    <h3>...</h3>
 </JobsPage>
);
```

**Comment by gabrielamatias** [comment]
@jesssummerfield , @petehamilton : please review.

**Pull request merged**